crypto/spiffe/trustanchors/static.go

```diff
@@ -28,22 +29,41 @@ import (
 // static is a TrustAcnhors implementation that uses a static list of trust
 // anchors.
 type static struct {
-	bundle  *x509bundle.Bundle
-	anchors []byte
-	running atomic.Bool
-	closeCh chan struct{}
+	x509Bundle *x509bundle.Bundle
+	jwtBundle  *jwtbundle.Bundle
+	anchors    []byte
+	running    atomic.Bool
+	closeCh    chan struct{}
 }
 
-func FromStatic(anchors []byte) (Interface, error) {
-	trustAnchorCerts, err := pem.DecodePEMCertificates(anchors)
+type OptionsStatic struct {
+	Anchors []byte
+	Jwks    []byte
+}
```

Comment:
Consider creating sub packages `trustanchors/static` and `trustanchors/multi` to clean up func and options prefix naming.